tests/unit/plugins/module_utils/test_system_access_users_utils.py

```diff
@@ -685,11 +841,9 @@ def test_generate_hashed_secret_failure_invalid_hash(mock_run_function):
         "stdout": "$5$somerandomsalt$shortvalue",
         "stderr": None,
     }
-
     user = User(name="test", password="test")
     with pytest.raises(OPNSenseCryptReturnError) as excinfo:
```

Comment:
```suggestion
    with pytest.raises(OPNsenseCryptReturnError) as excinfo:
```

Reply:
Thanks for the suggestion, fixed in cfcd2b4439517a3f4a6370ccb6f56fa95486db52.